index.html

```diff
@@ -58,6 +59,213 @@ <h1 class="identity_text">I'm Simba, I'm glad to see you!</h1>
+        </li>
+        <li class="list_items">
+          <p class="list_items_text_other">2015</p>
+        </li>
+      </ul>
+      <p class="project_description">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+      <ul class="technology_used">
+        <li class="technology">
+          <p class="technology_text">html</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">css</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">javascript</p>
+        </li>
+      </ul>
+      <button class="project_button" type="submit">See Project</button>
+    </div>
+  </section>
+
+  <section class="aboutme_container">
+    <div class="aboutme_class">
+      <h1 class="identity_text">About Myself</h1>
+      <p class="description_text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <p class="connect_text">LET'S CONNECT</p>
+      
+      <ul class="social_media_icons">
+        <li>
+          <a>
+          <img src="twitter.png" class="icons" alt="twitter">
+          </a>
+        </li>
+        <li>
+          <a>
+          <img src="github.png" class="icons" alt="github">
+        </a>
+        </li>
+        <li>
+          <a>
+          <img src="Linkedin_icon.png" class="icons" alt="linkedin">
+          </a>
+        </li>
+        <li>
+          <a>
+          <img src="medium.png" class="icons" alt="medium">
+          </a>
+        </li>
+        <li>
+          <a>
+          <img src="angellist.png" class="icons" alt="angellist">
+          </a>
+        </li>
+      </ul>
+      <button class="resume_button" type="submit">Get my resume</button>
+      <ul class="technology_and_skills">
+        <li class="technology_and_skills_item">
+          <h1 class="technology_and_skills_text">Languages</h1>
+          <ul class="languages">
+            <li class="language_list">
+              <img src="javascript.png" alt="javascript" class="language_image">
+              <p class="language_text">Javascript</p>
+            </li>
+            <li class="language_list">
+              <img src="html.png" alt="html" class="language_image">
+              <p class="language_text">HTML</p>
+            </li>
+            <li class="language_list">
+              <img src="css.png" alt="css" class="language_image">
+              <p class="language_text">CSS</p>
+            </li>
+          </ul>
+        </li>
+        <li class="technology_and_skills_item">
+          <h1 class="technology_and_skills_text">Frameworks</h1>
+          <hr>
+        </li>
+        <li class="technology_and_skills_item">
+          <h1 class="technology_and_skills_text">Skills</h1>
+          <hr>
+        </li>
```

Comment:
- [ ] Please make sure to follow Figma design in this part 
your version: 
![image](https://user-images.githubusercontent.com/74744787/159058982-94a4f2e1-6008-4f19-be72-3da23842ff2b.png)

figma design : 
![image](https://user-images.githubusercontent.com/74744787/159059054-3c72984f-fdbd-4adc-841c-2e5715aee7e5.png)